### PR TITLE
Registration page

### PR DESCRIPTION
**Pull request by Manikandan-Hub**
@nishintn 
@aswathym 

Hi,
Please find the pull request for the branch **registrationPage** which has the registration form and form validations.

hosted url: http://manikandan-hub.github.io/MebookWebApplicaton

### CODE REVIEW

File: js/registration.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+document.getElementById('regButton').addEventListener("click", validate);	
+});
+
+function validate() {
+    validatePasswordMatching();
+    validateDropDownMenus();
+}
+
+function validateTextField(element) {
+	console.log("Entered into function");
```
Comment (nishintn) [inline js/registration.js]: Get rid of console stms from remote branches.

File: js/registration.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
+
+window.addEventListener("load",function() {
+    console.log("Registration page loaded");
+
+document.getElementById('regButton').addEventListener("click", validate);
```
Comment (nishintn) [inline js/registration.js]: Store this on to a var and then add listener to it.

File: js/registration.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+function validateTextField(element) {
+	console.log("Entered into function");
+   if (element.value == '')
+   {
+       element.style.border = '1px solid red';
+       console.log("Empty field");
+   }
+   else
+   {
+   	console.log("have text");
```
Comment (nishintn) [inline js/registration.js]: Use console only for debugging.

File: js/registration.js [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+        console.log('not matches');
+        mobileNumber.setCustomValidity('Please provide a valid mobile number');
+    }
+}
+
+function emailValidation(emailId) {
+    var numberPattern =/^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    if (emailId.value.match(numberPattern)) {
+        emailId.setCustomValidity('');
+        console.log('matches');
```
Comment (nishintn) [inline js/registration.js]: Use console only fir debugging.

File: js/registration.js [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    }
+    else
+    {
+        console.log('not matches');
```
Comment (nishintn) [inline js/registration.js]: Use console only fir debugging.

**Comment by nishintn** [comment]
@Manikandan-Hub ,
Added some feedbacks.

**Comment by Manikandan-Hub** [comment]
@nishintn 
Hi,
Changes made in js documents as you suggested.
- Removed console log statements.

**Pull request merged**